### ISSUE TITLE
case-sensitive filename issues

### ISSUE THREAD

**Issue by tipsy** [tipsy/j2html#163]
```
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'src/main/java/j2html/tags/attributes/IFormAction.java'
  'src/main/java/j2html/tags/attributes/IFormaction.java'
  'src/main/java/j2html/tags/specialized/TextAreaTag.java'
  'src/main/java/j2html/tags/specialized/TextareaTag.java'
```

**Comment by pointbazaar** [comment]
Wow, a huge mistake i made there !
2 are probably remaining from a previous code_gen run and are out of date.

So 2 would need to be removed (the ones which implement fewer interfaces probably).

So sorry i didn't look at this issue sooner! I'll try to make a PR.

**Comment by pointbazaar** [comment]
[PR](https://github.com/tipsy/j2html/pull/167) is out!

**Comment by pointbazaar** [comment]
I think this issue has been resolved, would you mind closing the issue?

**Comment by tipsy** [comment]
Closed, thanks for reminding me! In the future you can add something like `(closes #163)` to the PR title, and the issue will be automtically closed when the PR is merged :)
